spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/support/ArgumentSanitizer.java

```diff
@@ -18,13 +18,17 @@
 
 import java.util.regex.Pattern;
 
+import org.springframework.cloud.dataflow.core.dsl.ArgumentNode;
+
 /**
  * Sanitizes potentially sensitive keys for a specific command line arg.
  * @author Glenn Renfro
  */
 public class ArgumentSanitizer {
```

Comment:
Do you happen to know why the boot guys didn't make this class public

https://pivotal.slack.com/archives/G0M0F86AJ/p1497984170701058